Source/SIMPLib/Common/Observer.h

```diff
@@ -54,8 +54,18 @@ class SIMPLib_EXPORT Observer : public QObject, public IObserver
 {
   Q_OBJECT
 
+   // Start Python bindings declarations
+  PYB11_BEGIN_BINDINGS(Observer)
+  PYB11_SHARED_POINTERS(Observer)
+  PYB11_CREATION()
+  PYB11_END_BINDINGS()
+  // End Python bindings declarations
+
 public:
   Observer();
+
+  Observer(QObject* parent);
+
```

Comment:
I don't think we need this change anymore since we went with the new API in FilterPipeline.